ci/taos/plugins-base/pr-audit-build-android.sh

```diff
@@ -168,6 +185,42 @@ function pr-audit-build-android-run-queue(){
         popd
         echo -e "[DEBUG] The current directory: $(pwd)."
 
+        # Start to build NNStreamer API for Android.
+        echo "[DEBUG] Starting gradle build for Android API."
+
+        # NNStreamer root directory for build.
+        export NNSTREAMER_ROOT=$(pwd)
+
+        # Make directory to build NNStreamer library.
+        mkdir -p build_android_lib
+
+        # Copy the files (native and java to build Android library) to build directory.
+        cp -r ./api/android/* ./build_android_lib
+
+        # Get the prebuilt libraries and build-script.
+        cp -r /var/www/html/android/nnstreamer-android-api/* ./build_android_lib
```

Comment:
Could you append the variable $ROOT_ANDROID_CI  to remove duplicated folder names such as "/var/www/html/android"?

For example, 
```bash
ROOT_ANDROID_CI="/var/www/html/android"
cp -r ${ROOT_ANDROID_CI}/nnstreamer-android-api/* ./build_android_lib
```

Reply:
As you commented, I added ROOT_ANDROID_CI and changed common path.